src/bin/vip-import-media.js

```diff
@@ -66,15 +92,66 @@ command( {
 	appContext: true,
 	appQuery,
 	envContext: true,
-	requiredArgs: 1,
 	module: 'import-media',
-	requireConfirm: 'Are you sure you want to import the contents of the provided file?',
-	skipConfirmPrompt: true,
+	requireConfirm: 'Are you sure you want to import the contents of the url?',
 } )
-	.command( 'status', 'Check the status of the current running import' )
+	// .command( 'status', 'Check the status of the current running import' )
+	.option( 'url', 'Valid URL to download a file archive from', '' )
 	.examples( examples )
 	.argv( process.argv, async ( args: string[], opts ) => {
+		const { app, env, url } = opts;
+
+		if ( ! url ) {
+			console.log( chalk.red( 'Error:' ), 'No URL provided' );
+			return;
+		}
+
+		const track = trackEventWithEnv.bind( null, app.id, env.id );
+		const api = await API();
+
 		debug( 'Options: ', opts );
 		debug( 'Args:', args );
+
+		await track( 'import_media_start_execute' );
+
+		const progressTracker = new MediaImportProgressTracker( [] );
+		progressTracker.prefix = `
+=============================================================
+Processing the files import for your environment...
+`;
+
+		console.log();
+		console.log( `importing: ${ url }` );
+		console.log( `to: ${ env.primaryDomain.name }` );
```

Comment:
I think maybe, this line and the one after it can be somehow combined? they offer the same information

Reply:
Maybe just a `to: ${ env.primaryDomain.name } (${ env.type})`?
We are already showing the env name in the status checks so I'm thinking we can just remove that. What do you think?

Reply:
thanks! looks good!